app/addons/documents/changes/actions.js

```diff
@@ -38,11 +45,59 @@ function (app, FauxtonAPI, ActionTypes) {
       });
     },
 
-    setChanges: function (options) {
+    initChanges: function (options) {
       FauxtonAPI.dispatch({
-        type: ActionTypes.SET_CHANGES,
+        type: ActionTypes.INIT_CHANGES,
         options: options
       });
+      this.getLatestChanges();
+    },
+
+    getLatestChanges: function () {
+      var params = {};
+
+      var lastSeqNum = changesStore.getLastSeqNum();
```

Comment:
I just tried this, but for the initial request it can't be a `longpoll` or I don't ever get the first batch of data returned. But I like using 'now' as a default - much cleaner - so I refactored the code a bit to only set it to longpoll once `currentRequest` has been set. Bit cleaner. [Will commit in a moment]